home.html

```diff
@@ -148,25 +114,16 @@
 
 
 
-        <!-- Page Content -->
+        <!-- Page Content:Hero Section -->
         <div class="pt-20">
             <div class="container mx-auto text-center text-white">
-                <h1 class="text-5xl font-bold mt-24 uppercase">Welcome to Our <span class="text-red-900">Website</span></h1>
-                <div class="flex justify-between gap-6 place-items-center py-6 sm">
-                    <div class="text-white bg-orange-600 rounded-2xl bg-opacity-40 text-2xl sm:text-3xl md:text-4xl lg:text-5xl p-5 md:h-auto">
-                        <span>Where Tradition Meets Taste</span>
-                    </div>
-                    <div class="text-white bg-orange-600 rounded-2xl bg-opacity-40 text-2xl sm:text-3xl md:text-4xl lg:text-5xl p-5 md:h-auto">
-                        <span>Discounts are assured on all ordered dishes</span>
-                    </div>
-                </div>
-
+                <h1 class="text-5xl font-bold mt-24 mb-8 uppercase">Welcome to Our <span class="text-red-900">Website</span></h1>
                 <div class="relative overflow-hidden">
                     <div id="slider" class="flex transition-transform duration-500 ease-in-out">
                       <!-- Slide 1 -->
                       <div class="bg-white flex-shrink-0 w-full flex">
                         <div class="block h-full w-2/5 ml-20">
-                          <img src="logo.png" alt="logo">
+                          <img src="/images/background/logo.png" alt="logo">
```

Comment:
No need to use abi's kitchen logo on your hero. You already have it on your navbar. Kindly take it out.